### PR TITLE
Update series list for local charm deployments.

### PR DESCRIPTION
**Pull request by frankban**
It would be nice to include this new series in the next charm release as some of them are required in a demo at the end of the week.

### CODE REVIEW

File: app/store/env/go.js [hunk 1]
```diff
@@ -155,7 +155,10 @@ YUI.add('juju-env-go', function(Y) {
         To be updated as necessary.
       @type {Array}
     */
-    series: ['precise', 'quantal', 'raring', 'saucy', 'trusty'],
+    series: [
+      'precise', 'quantal', 'raring', 'saucy', 'trusty', 'utopic', 'vivid',
+      'win2012hvr2', 'win2012hv', 'win2012r2', 'win2012', 'win7', 'win8',
+      'win81'],
```
Comment (urosj) [inline app/store/env/go.js]: I'm not sure if I (or anyone) like the cryptic series. It's a start, but I'd consider mapping those to proper names.

**Comment by urosj** [comment]
:+1: for a start, leave mapping for future PR

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2706/
Test FAILed.

**Comment by frankban** [comment]
CI failure seems spurious.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**